demo/allegro5/nuklear_allegro5.h

```diff
@@ -111,11 +111,11 @@ nk_allegro5_font_get_text_width(nk_handle handle, float height, const char *text
        as nuklear uses variable size buffers and al_get_text_width doesn't
        accept a length, it infers length from null-termination
        (which is unsafe API design by allegro devs!) */
-    strcpy = malloc(len + 1);
-    strncpy(strcpy, text, len);
-    strcpy[len] = '\0';
-    width = al_get_text_width(font->font, strcpy);
-    free(strcpy);
+    str = calloc((size_t)len + 1, 1);
```

Comment:
When at it, shouldn't the last `1` be actually `sizeof( *str )`?

Reply:
Or `sizeof(char)` ?

Reply:
I believe `sizeof(char) != 1` is true only on weird architectures that went extinct decades ago. I'll change that if you insist though :slightly_smiling_face:

Reply:
Just leave things as they are. I had yet another nit pick day, sorry for that :wink:.

Reply:
No problem. Thanks for your dedication :blush: